packages/rs-dpp/src/data_contract/state_transition/data_contract_update_transition/validation/basic/validate_data_contract_update_transition_basic.rs

```diff
@@ -201,6 +205,23 @@ where
             }
         }
 
+        if !validation_result.is_valid() {
```

Comment:
Just curious. Has the change of validation order fixed some tests?

Reply:
Exactly! Otherwise indices validator will bail on missing properties that should exist for validation process rather than consensus error